src/playkit.js

```diff
@@ -26,6 +27,9 @@ export {registerMediaSourceAdapter};
 // Export the plugin framework
 export {registerPlugin, BasePlugin};
 
+// Export the tracks classes
+export {TRACK};
```

Comment:
Capital letters convention point on constant vars (like the VERSION) and here it's not the case. I think it better to export it regularly...
`export {AudioTrack, TextTrack, VideoTrack....}` and so on